src/border_agent/border_agent.cpp

```diff
@@ -309,8 +309,20 @@ void BorderAgent::PublishMeshCopService(void)
 
     mPublisher->PublishService(/* aHostName */ "", mServiceInstanceName, kBorderAgentServiceType,
                                Mdns::Publisher::SubTypeList{}, port, txtList, [this](otbrError aError) {
-                                   otbrLogResult(aError, "Result of publish meshcop service %s.%s.local",
-                                                 mServiceInstanceName.c_str(), kBorderAgentServiceType);
+                                   if (aError == OTBR_ERROR_ABORTED)
+                                   {
+                                       // OTBR_ERROR_ABORTED is thrown when an ongoing service registration is
+                                       // gracefully cancelled. This can happen when the meshcop service is being
+                                       // updated frequently. To avoid false alarms, it should not be logged like a real
+                                       // error.
+                                       otbrLogInfo("Cancel publishing meshcop service %s.%s.local",
+                                                   mServiceInstanceName.c_str(), kBorderAgentServiceType);
+                                   }
+                                   else
+                                   {
+                                       otbrLogResult(aError, "Result of publish meshcop service %s.%s.local",
```

Comment:
What about putting the logic into `otbrLogResult` and dump `NOTE` level messages for all `OTBR_ERROR_ABORTED` errors?

Reply:
`ABORT` means the operation is explicitly cancelled by the client, so maybe all such errors are actually not an error.

Reply:
> `ABORT` means the operation is explicitly cancelled by the client, so maybe all such errors are actually not an error.

I think it might be too aggressive. Maybe we can do it later when we have more signals.

Reply:
No strong opinion.